includes/discovery/sensors/frequency/linux.inc.php

```diff
@@ -3,24 +3,22 @@
  * raspberry pi frequencies
  * requires snmp extend agent script from librenms-agent
  */
-$raspberry = snmp_get($device, 'HOST-RESOURCES-MIB::hrSystemInitialLoadParameters.0', '-Osqnv');
+$sensor_type = "raspberry_freq";
+$oid = '.1.3.6.1.4.1.8072.1.3.2.4.1.2.9.114.97.115.112.98.101.114.114.121.';
```

Comment:
That would mean walking the entire table all the time when you only need a couple of values.

I can try and get this walk into pre-cache but it's not a named OID so array returned using numerical OIDs are horrible.